### PR TITLE
Update article for stopping systemd in Openshift

### PR DESCRIPTION
**Pull request by avisiedo**
- The situation was tried in Openshift 4.10 and it fixes
  the situation.
- Update the script which is used for testing the signals.
- Some mistakes detected during updating the article, such as
  imagePullPolicy attribute that was not specified, and the
  validation for SIGINT which indeed is working in Openshift.
- Update .gitignore for .venv and output directories.

### CODE REVIEW

File: content/stopping-systemd-in-openshift.md [hunk 1]
```diff
@@ -24,9 +23,15 @@ limitation in Kubernetes by using container lifecycle hooks.
 > [kcli](https://github.com/karmab/kcli) or
 > [Code Ready Containers](https://github.com/code-ready/crc).
 
+**UPDATES**:
+
+- This is happening in Openshift but it will be fiex inin 4.10 (verified on
```
Comment (frasertweedale) [inline content/stopping-systemd-in-openshift.md]: fiex -> fixed

File: content/stopping-systemd-in-openshift.md [hunk 2]
```diff
@@ -24,9 +23,15 @@ limitation in Kubernetes by using container lifecycle hooks.
...
+  Openshift 4.10.0-ci-20220107).
+- Here is the change at cri-o that fix this situation:
```
Comment (frasertweedale) [inline content/stopping-systemd-in-openshift.md:30]: missing link?

File: content/stopping-systemd-in-openshift.md [hunk 3]
```diff
@@ -174,91 +182,114 @@ Detected architecture x86-64.
...
 FROM quay.io/fedora/fedora:35
 COPY demo-signal.sh /demo-signal.sh
+RUN chmod a+x /demo-signal.sh
 STOPSIGNAL SIGINT
-ENTRYPOINT ["/demo-signal.sh"]
+CMD ["/demo-signal.sh"]
 ```
 
-The `demo-signal.sh` should have execute permission.  The content is:
+The `demo-signal.sh` should have executing permission. The content is:
```
Comment (frasertweedale) [inline content/stopping-systemd-in-openshift.md]: I think the previous text "execute permission" is more natural.

**Review by frasertweedale — COMMENTED** [review]
Minor style/spelling comments.

File: content/stopping-systemd-in-openshift.md (hunk 1 above)
Comment (avisiedo) [reply content/stopping-systemd-in-openshift.md]: Thanks!

File: content/stopping-systemd-in-openshift.md (hunk 2 above)
Comment (avisiedo) [reply content/stopping-systemd-in-openshift.md:30]: It should be https://github.com/cri-o/cri-o/pull/5366 ; updating, thanks

File: content/stopping-systemd-in-openshift.md (hunk 3 above)
Comment (avisiedo) [reply content/stopping-systemd-in-openshift.md]: thanks!

**Pull request merged**